_posts/2015-10-08-expected-behaviors.md

```diff
@@ -16,4 +16,6 @@ Every member of the product team is expected to work hard, be considerate of the
 
 **Be respectful toward remote and IRL interactions alike.** Every member of our team is remote at least some of the time. And with two main offices and several smaller offices, even those of us who work in an office nearly every day are routinely remote to others. Adopt habits that are inclusive and productive for team members wherever they are: make liberal use of video hangouts, document meetings and decisions thoroughly, and pay attention to timezones when scheduling events. 
 
-**Be humane.** Be polite and friendly in all forms of communication, especially remote communication, where opportunities for misunderstanding are greater. Use sarcasm carefully. Tone is hard to decipher online; make judicious use of emoji to aid in communication. Use video hangouts and IRL meetings when it makes sense; face-to-face discussion benefits from all kinds of social cues that may go missing in other forms of communication. 
+**Be humane.** Be polite and friendly in all forms of communication, especially remote communication, where opportunities for misunderstanding are greater. Use sarcasm carefully. Tone is hard to decipher online; make judicious use of emoji to aid in communication. Use video hangouts and IRL meetings when it makes sense; face-to-face discussion benefits from all kinds of social cues that may go missing in other forms of communication.
+
+**Treat partners and third parties with the same respect you would a coworker.** We work with outside parties from time to time, some of whom may not be subject to this code of conduct. Internal team discussion about external parties should reflect the same respect, constructive and humane nature that we strive for in our own teams. 
```

Comment:
The only change I'd suggest is replacing "partners" with "customers" to match our internal vocabulary. Otherwise, :+1: